measurement_files/geography_based.py

```diff
@@ -48,6 +52,66 @@ def geo_ctr(dataloader, args):
 
     pickle.dump(counts, open("results/{}/geo_ctr.pkl".format(args.folder), "wb"))
 
+def custom_geo_ctr(dataloader, args):
+    # TODO: remove process_cutoff once finalized
```

Comment:
is this finalized enough to remove?

Reply:
still doing a bit of testing, i will remove it after i get my new modifications in and will keep this comment as a reminder for myself.